cpanfile

```diff
@@ -1,12 +1,12 @@
 requires 'IO::Socket';
-requires 'Locale::Country';
+requires 'Locale::Country', '3.66';
```

Comment:
Locale::Country is removed from perl distribution. So there will warn to tell use we should install it from cpan.
Giving it a version number here will tell dzil to install it from cpan.

Reply:
We are using v3.66 in our codes. So here it is changed to 3.66